src/main/java/seedu/address/logic/commands/FilterCommand.java

```diff
@@ -27,9 +27,11 @@ public class FilterCommand extends UndoableCommand {
             + "Parameters: [s/RSVPSTATUS] [t/TAG]...\n" + "At least one parameter must be provided.\n"
             + "Example: " + COMMAND_WORD + " s/1 t/bride's side";
     public static final String MESSAGE_TAG_NOT_CREATED = "Tag(s) must be created first using 'newtag' command: ";
-    public static final String MESSAGE_TAG_FILTER_ALREADY_EXISTS = "The following tags are already being filtered: ";
-    public static final String MESSAGE_STATUS_ALREADY_BEING_FILTERED = "The following statuses are not filtered "
-            + "as a status filter is already applied: ";
+    public static final String MESSAGE_TAG_FILTER_ALREADY_EXISTS = "The following tag(s) are already being filtered: ";
+    public static final String MESSAGE_EXISTING_STATUS_FILTERED = "The following RSVP status(es) are not filtered "
```

Comment:
Should there be a colon here to standardize the 2 messages?

Reply:
The message has not ended yet. There is a colon below.